sale_order_color_code_tintometric_system_stock/reports/stock_picking_template.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <template id="report_delivery_document" inherit_id="stock.report_delivery_document">
+        <xpath expr="//th[@name='th_sm_product']" position="after">
+            <t t-set="display_color" t-value="any(line.color_code_tintometric for line in o.move_ids_without_package)"/>
+            <t t-if="display_color">
+                <th name="color_code_tintometric"><strong>Lot/S.Tin</strong></th>
+            </t>
+        </xpath>
+        <xpath expr="//span[@t-field='move.product_uom_qty']/.." position="before">
+            <t t-set="display_color" t-value="any(line.color_code_tintometric for line in o.move_ids_without_package)"/>
```

Comment:
`display_color` variable should be available at this point, so this line should be unnecessary